test/any/files-helper.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+// Manually require code under test (in alphabetical order).
+//
+// TODO(markdittmer): Move to FOAM classloader for this.
+
+beforeAll(function() {
+  // Use of foam.isServer = <running in Node JS?>:
+  // - Do not load Angular-related in Node JS (foam.isServer = true).
+  // - Do not load import-from-filesystem in browser.
+
+  if (!foam.isServer) {
+    // TODO(markittmer): Fix: Alphabetical order exception: main/app defines
+    // angular module used by other code.
```

Comment:
Is the angular module only required by stuff imported directly after this? If it is, document that the dependent code is entirely in this block. If not, I think it would be better to break the overall alphabetical ordering and move those imports here, and then document it.